src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/EntityComponent.scala

```diff
@@ -35,36 +37,80 @@ trait EntityComponent {
     type EntityQuery = Query[EntityTable, EntityRecord, Seq]
     type EntityQueryWithAttributesAndRefs =  Query[(EntityTable, Rep[Option[(EntityAttributeTable, Rep[Option[EntityTable]])]]), (EntityRecord, Option[(EntityAttributeRecord, Option[EntityRecord])]), Seq]
 
-    implicit val getEntityRecord = GetResult { r => EntityRecord(r.<<, r.<<, r.<<, r.<<, r.<<) }
+    private object EntityRecordRawSqlQuery extends RawSqlQuery {
```

Comment:
this is a simple example of the new RawSqlQuery pattern. It has a a GetResult implicit. It reuses the EntityRecord case class for the result instead of defining a custom one. It has an action method that takes the parameters specific to the query and produces the sql action.